docs/ops.html

```diff
@@ -935,6 +974,289 @@
+  <tbody>
+    <tr>
+      <th>Metric/Attribute name</th>
+      <th>Description</th>
+      <th>Mbean name</th>
+    </tr>
+    <tr>
+      <td>fetch-size-avg</td>
+      <td>The average number of bytes fetched per request for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-size-max</td>
+      <td>The maximum number of bytes fetched per request for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>bytes-consumed-rate</td>
+      <td>The average number of bytes consumed per second for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>records-per-request-avg</td>
+      <td>The average number of records in each request for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>records-consumed-rate</td>
+      <td>The average number of records consumed per second for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+  </tbody>
+</table>
+
+<h5><a id="consumer_global_request_monitoring" href="#consumer_global_request_monitoring">Global Request Metrics</a></h5>
+
+<table class="data-table">
+  <tbody>
+    <tr>
+      <th>Metric/Attribute name</th>
+      <th>Description</th>
+      <th>Mbean name</th>
+    </tr>
+    <tr>
+      <td>request-latency-avg</td>
+      <td>The average request latency in ms.</td>
+      <td>kafka.consumer:type=consumer-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>request-latency-max</td>
+      <td>The maximum request latency in ms.</td>
+      <td>kafka.consumer:type=consumer-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>request-rate</td>
+      <td>The average number of requests sent per second.</td>
+      <td>kafka.consumer:type=consumer-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>response-rate</td>
+      <td>The average number of responses received per second.</td>
+      <td>kafka.consumer:type=consumer-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>incoming-byte-rate</td>
+      <td>The average number of incoming bytes received per second from all servers.</td>
+      <td>kafka.consumer:type=consumer-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>outgoing-byte-rate</td>
+      <td>The average number of outgoing bytes sent per second to all servers.</td>
+      <td>kafka.consumer:type=consumer-metrics,client-id=([-.\w]+)</td>
+    </tr>
+  </tbody>
+</table>
+
+<h5><a id="consumer_global_connection_monitoring" href="#consumer_global_connection_monitoring">Global Connection Metrics</a></h5>
```

Comment:
These metrics seem to already be covered in the "Common monitoring metrics" section above.